corehq/apps/accounting/invoicing.py

```diff
@@ -794,6 +797,14 @@ def unit_description(self):
         return self._unit_description_by_user_type("mobile user")
 
 
+class FormSubmittingMobileWorkerLineItemFactory(UserLineItemFactory):
+    history_cls = FormSubmittingMobileWorkerHistory
+
+    @property
+    def unit_description(self):
+        return super()._unit_description_by_user_type("form-submitting mobile worker")
```

Comment:
> 1. `UserLineItemFactory` uses "mobile user", whereas we are using "mobile worker" here. Are they referring to different things, or should we standardize?

"User" and "mobile worker" here are the same thing. I'll check in with ops and see what they think about naming on invoices. They also had some lack of clarity about where/when the distinction between "web _user_" and "mobile _worker_" came about.


> 2. This appears to be a problem with `UserLineItemFactory` as well, but `_unit_description_by_user_type` translates its strings. Because the `user_type` is only injected after translation, it looks like "form-submitting mobile worker" (and "mobile user") will not get translated. I'd imagine these strings should be translated.

I think you're probably right, but again I'll check with ops since changing it on existing features would apply across all translated invoices. Would that just be a matter of restructuring this `ngettext` block? https://github.com/dimagi/commcare-hq/blob/cb08c50fa38b142b2657b1e5bfcded89916c904c/corehq/apps/accounting/invoicing.py#L780-L789

Reply:
There remains a small debate over naming that I'm waiting to hear on - ops would prefer to see "form-submitting mobile _user_" for consistency, but otherwise I think this is addressed.